deploy/manifests/base/storetheindex/kustomization.yaml

```diff
@@ -3,11 +3,25 @@ kind: Kustomization
 
 namespace: storetheindex
 
+resources:
+  - indexer-statefulset.yaml
+  - finder-service.yaml
+  - ingest-service.yaml
+
 transformers:
   - labels.yaml
 
 configMapGenerator:
-  - name: storetheindex-config
+  - name: indexer-env-vars
     behavior: create
     literals:
       - GOLOG_LOG_LEVEL=INFO
+      - GOLOG_LOG_FMT=json
+      - STORETHEINDEX_LOTUS_GATEWAY=wss://api.chain.love
+      - STORETHEINDEX_PATH=/config
+      - STORETHEINDEX_LISTEN_ADMIN=/ip4/0.0.0.0/tcp/3002
```

Comment:
Is this used? I thought this was set in the config?

Reply:
Added it here since it overrides the default. It also gets overridden in docker file for some reason. and yes it is also set in config.  I can't remember which gets picked up in the end. so I set them. happy to remove redundant ones.

Reply:
I'd say yes to remove the redundant ones, since it's not clear which ones get set (env should always have priority fwiw).

> since it overrides the default.

Isn't 3002 the default anyways?

Reply:
3002 yes but not `0.0.0.0`.

Reply:
ah I see. Thank you